ports/psoc6/boards/CY8CPROTO-062-4343W/.cyignore

```diff
@@ -0,0 +1,35 @@
+# Documentation
```

Comment:
Is this automatically added when setting up MTB ?

Reply:
no, this cy ignore file needs to be added manually.